scripts/cluster.sh

```
@@ -106,6 +145,7 @@ function check_create_cluster() {
                 -P pull_secret="${OPENSHIFT_PULL_SECRET}" \
                 -P high_availability_mode=None \
                 -P user_managed_networking=True \
+		$([ "${NODES_MTU}" != "1500" ] && echo "--paramfile ${STATIC_NET_FILE}") \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)